Button1: 0
Button2: 1
Button3: 2
Button4: 3
Button5: 4
Button6: 5
Button7: 6
Button8: 7
Button11: 10
Button12: 11
AxisLeftStickX: 0
AxisLeftStickY: 1
AxisRightStickX: 2
AxisRightStickY: 3
AxisJuziX: 4
AxisJuziY: 5
